### PR TITLE
Fix DRC standalone command

### PR DESCRIPTION
**Pull request by osamahammad21**
- Fix io reading from wire encoder
- Fix initialization of gcWorker nets from design directly (no drWorker)
- add drc_test

### CODE REVIEW

File: src/drt/src/io/io.cpp [hunk 1]
```diff
@@ -750,8 +750,6 @@ void io::Parser::setNets(odb::dbBlock* block)
               nextX = beginX;
               nextY = beginY;
             }
-            prevLayer = decoder.getLayer();
-            layerName = prevLayer->getName();
```
Comment (osamahammad21) [inline src/drt/src/io/io.cpp:754]: @eder-matheus this part was added in your [PR](https://github.com/The-OpenROAD-Project/OpenROAD/pull/3411) previously which messed up the layer assignment of the read segments from wire encoder. I reran the test case that your PR should solve and it ran through without any problem.
Could you please confirm that this does not conflict with your initial fix?

**Comment by maliberty** [comment]
unit test failure

**Comment by osamahammad21** [comment]
> unit test failure

fixed

File: src/drt/src/io/io.cpp (hunk 1 above)
Comment (eder-matheus) [reply src/drt/src/io/io.cpp:754]: From what I reviewed, it doesn't conflict with my changes.

**Review by maliberty — APPROVED** [review]

**Pull request merged**